e2e/test/prerequisites/E2ETestsSetup/e2eTestsSetup.ps1

```diff
@@ -496,10 +502,69 @@ $iotHubCertChainDevice = az iot hub device-identity list -g $ResourceGroup --hub
 
 if (-not $iotHubCertChainDevice)
 {
-    Write-Host "`nCreating device $iotHubCertChainDeviceCommonName on IoT Hub."
+    Write-Host "`nCreating X509 CA certificate authenticated device $iotHubCertChainDeviceCommonName on IoT Hub."
     az iot hub device-identity create -g $ResourceGroup --hub-name $iotHubName --device-id $iotHubCertChainDeviceCommonName --am x509_ca
 }
 
+##################################################################################################################################
+# Create the IoT devices and modules that are used by the .NET samples
+##################################################################################################################################
+$iotHubSasBasedDeviceId = "DoNotDeleteDevice1"
```

Comment:
Since these are only devices and enrollments, I didn't place them under the `GenerateResourcesForDevOpsPipeline` flag.